ansible/roles/master/templates/apiserver.j2

```diff
@@ -23,4 +23,4 @@ KUBE_ETCD_SERVERS="--etcd-servers={% for node in groups['etcd'] %}{% if etcd_url
 KUBE_ADMISSION_CONTROL="--admission-control={{ admission_controllers }}"
 
 # Add your own!
-KUBE_API_ARGS="{{ apiserver_extra_args | default('') }} --tls-cert-file={{ kube_cert_dir }}/server.crt --tls-private-key-file={{ kube_cert_dir }}/server.key --client-ca-file={{ kube_cert_dir }}/ca.crt --token-auth-file={{ kube_token_dir }}/known_tokens.csv --service-account-key-file={{ kube_cert_dir }}/server.crt --bind-address={{ kube_apiserver_bind_address }}"
+KUBE_API_ARGS="--tls-cert-file={{ kube_cert_dir }}/server.crt --tls-private-key-file={{ kube_cert_dir }}/server.key --client-ca-file={{ kube_cert_dir }}/ca.crt --token-auth-file={{ kube_token_dir }}/known_tokens.csv --service-account-key-file={{ kube_cert_dir }}/server.crt --bind-address={{ kube_apiserver_bind_address }} {{ kube_apiserver_options|join(' ') }}"
```

Comment:
Given ``kube_cert_dir`` [1], ``kube_token_dir`` [2] are defined in ``kubernetes`` role and other variables used in defining options are known in ``master/defaults/main.yml``, think it would make more sense to change the line to:

```sh
KUBE_API_ARGS="{{ kube_apiserver_options|join(' ') }}"
```
and move all the options into the master default vars?
```yml
kube_apiserver_options:
  - "--tls-cert-file={{ kube_cert_dir }}/server.crt"
  - "--tls-private-key-file={{ kube_cert_dir }}/server.key"
  - "--client-ca-file={{ kube_cert_dir }}/ca.crt"
  - " --token-auth-file={{ kube_token_dir }}/known_tokens.csv"
  - "--service-account-key-file={{ kube_cert_dir }}/server.crt"
  - "--bind-address={{ kube_apiserver_bind_address }}"
```

So if the variable is named ``kube_apiserver_options`` it has all options defined at one place.

At the same time do the same for other ``kube_*_options`` variables and templates?

[1] https://github.com/kubernetes/contrib/blob/master/ansible/roles/kubernetes/defaults/main.yml#L36
[2] https://github.com/kubernetes/contrib/blob/master/ansible/roles/kubernetes/defaults/main.yml#L42